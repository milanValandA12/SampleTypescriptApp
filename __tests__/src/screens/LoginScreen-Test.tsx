import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../../../src/redux/store';
import LoginScreen from '../../../src/screens/auth/login.screen';

describe('Login screen', () => {
    it('renders correctly', () => {
        const tree = renderer.create(
            <Provider store={store}>
                <LoginScreen />
            </Provider>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});