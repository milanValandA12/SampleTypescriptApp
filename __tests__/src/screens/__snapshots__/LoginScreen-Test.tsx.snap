// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login screen renders correctly 1`] = `
<View
  style={
    Object {
      "marginTop": 120,
      "padding": 8,
    }
  }
>
  <Text>
    login:form.label
  </Text>
  <TextInput
    onChangeText={[Function]}
    placeholder="login:form.label"
    style={
      Object {
        "borderColor": "#000000",
        "borderRadius": 12,
        "borderWidth": 1,
        "height": 50,
        "marginBottom": 12,
        "padding": 8,
      }
    }
    value=""
  />
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#000000",
        "borderRadius": 16,
        "height": 40,
        "justifyContent": "center",
        "margin": 4,
        "minWidth": 100,
        "opacity": 1,
        "padding": 8,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#ffffff",
        }
      }
    >
      login:getDetails
    </Text>
  </View>
</View>
`;
